src/routes/clients.js

```diff
@@ -79,14 +101,28 @@ router.post('/', (req, res) => {
   });
 });
 
+const normalizeClient = client => {
+  if (client.Permissions) {
+    client.dataValues.permissions = client.Permissions.map(
```

Comment:
Is it really needed to use `dataValues` here ? Can't you just assign to `client.permissions` ?